snow/engine/snowman/syncer/state_syncer.go

```diff
@@ -248,10 +248,10 @@ func (ss *stateSyncer) AcceptedStateSummary(ctx context.Context, nodeID ids.Node
 	ss.Ctx.Log.Debug("adding weight to summaries",
 		zap.Stringer("nodeID", nodeID),
 		zap.Stringer("subnetID", ss.Ctx.SubnetID),
-		zap.Stringers("summaryIDs", summaryIDs),
+		zap.Reflect("summaryIDs", summaryIDs),
```

Comment:
Hmm I've never seen `Reflect` before. I've always used `Any`, and the comment of `Reflect` says, in part, `Outside tests, Any is always a better choice.`

Reply:
ref https://pkg.go.dev/go.uber.org/zap#Reflect

Reply:
Didn't we use to have a `String() string` method on `set.Set`?

Reply:
`zap.Any` ends up using `zap.Reflect` internally... I feel like I prefer to be explicit here 🤷

Reply:
We have almost no usages of `zap.Any` in the codebase... So I think it makes sense to keep with `Reflect` (unless the concern is that we should use `Any` so that if a type changes in the future then that's handled at runtime... but ideally I feel like we'd change the usage of `Reflect` to the actual type that should be used